opencti-platform/opencti-front/src/private/components/settings/users/User.js

```diff
@@ -460,9 +460,9 @@ class UserComponent extends Component {
           container={true}
           spacing={3}
           classes={{ container: classes.gridContainer }}
-          style={{ marginTop: 20 }}
+          style={{ marginTop: 30 }}
         >
-          <Grid item={true} xs={12}>
+          <Grid item={true} xs={6} style={{ marginBottom: 25 }}>
```

Comment:
Why do we have this margin bottom?

Reply:
when passing History and Operations in the same line, there were a difference at the bottom (Operations didn't respect the margin bottom)